boards/witte/linum/linum.dts

```diff
@@ -21,72 +23,64 @@
 		zephyr,dtcm = &dtcm;
 		zephyr,code-partition = &slot0_partition;
 		zephyr,canbus = &fdcan1;
+		zephyr,display = &ltdc;
+		zephyr,flash-controller = &w25q128jv;
+		zephyr,touch = &ft5446;
 	};
 
-	sdram1: sdram@c0000000 {
-		compatible = "zephyr,memory-region", "mmio-sram";
+	sdram1:sdram@c0000000 {
+		compatible = "zephyr,memory-region","mmio-sram";
```

Comment:
Please move formatting changes to a dedicated commit.

Reply:
Ugh it happened when I passed the file through clang-format 🤦🏻‍♂️